webpack/module-minifier-plugin/src/ModuleMinifierPlugin.ts

```diff
@@ -535,6 +545,29 @@ export class ModuleMinifierPlugin implements webpack.Plugin {
           }
         );
 
+        const { usePortableModules, compressAsyncImports } = this._optionsForHash;
+
+        function updateChunkHash(hash: Hash, chunk: webpack.compilation.Chunk): void {
+          // For now, automatically assume that any change to this plugin will alter the hash.
+          hash.update(pluginVersion as string);
```

Comment:
Every time this plugin gets updated, all hashes will change. That seems suboptimal.